server/crawler/starlark_main_dot_star_parser.go

```diff
@@ -78,24 +81,48 @@ func parseExpr(rawArg syntax.Expr, extractedTypeFromComment *StarlarkArgumentTyp
 	switch typedArg := rawArg.(type) {
 	case *syntax.Ident:
 		return &StarlarkFunctionArgument{
-			Name:        typedArg.Name,
-			Description: "",
-			Type:        extractedTypeFromComment,
-			IsRequired:  true,
+			Name:         typedArg.Name,
+			Description:  "",
+			Type:         extractedTypeFromComment,
+			IsRequired:   true,
+			DefaultValue: nil,
 		}, nil
 	case *syntax.BinaryExpr:
-		parsedArgument, err := parseExpr(typedArg.X, extractedTypeFromComment)
+		parsedArgument, err := parseExpr(typedArg.X, extractedTypeFromComment) // X, left side of binary expr, is the arg name
 		if err != nil {
 			return nil, stacktrace.Propagate(err, "Unable to parse Starlark function argument: %v", typedArg)
 		}
+		parsedDefaultValue := parseDefaultValue(typedArg.Y) // Y, right side of binary expr, is the default value
 		return &StarlarkFunctionArgument{
-			Name:        parsedArgument.Name,
-			Description: "",
-			Type:        extractedTypeFromComment,
-			IsRequired:  false,
+			Name:         parsedArgument.Name,
+			Description:  "",
+			Type:         extractedTypeFromComment,
+			IsRequired:   false,
+			DefaultValue: parsedDefaultValue,
 		}, nil
 	default:
-		return nil, stacktrace.NewError("Type of function parameter no handled: %v", reflect.TypeOf(rawArg))
+		return nil, stacktrace.NewError("Type of function parameter not handled: %v", reflect.TypeOf(rawArg))
+	}
+}
+
+// Returns pointer to raw string of the default value if [rawDefaultValue] is a bool, int, or string
+// Returns nil otherwise
+func parseDefaultValue(rawDefaultValue syntax.Expr) *string {
+	var parsedDefaultValue string
+	switch typedDefaultValue := rawDefaultValue.(type) {
+	case *syntax.Ident: // True and False are Ident
+		if typedDefaultValue.Name == starlarkTrueValue || typedDefaultValue.Name == starlarkFalseValue {
+			parsedDefaultValue = strings.ReplaceAll(typedDefaultValue.Name, `"`, "") // strip quotes
+			return &parsedDefaultValue
+		} else {
+			return nil
+		}
+	case *syntax.Literal: // string and int are Literals
+		parsedDefaultValue = strings.ReplaceAll(typedDefaultValue.Raw, `"`, "") // strip quotes
+		return &parsedDefaultValue
+	default: // rawDefaultValue is not a primitive or supported yet
+		// TODO: add support for dict and list default values
```

Comment:
@tedim52 should we log a warning here? If a user tries to pass in a non primitive then we log it?